compiler/rustc_trait_selection/messages.ftl

```diff
@@ -60,3 +66,6 @@ trait_selection_unable_to_construct_constant_value = unable to construct a const
 
 trait_selection_unknown_format_parameter_for_on_unimplemented_attr = there is no parameter `{$argument_name}` on trait `{$trait_name}`
     .help = expect either a generic argument name or {"`{Self}`"} as format argument
+
+trait_selection_wrapped_parser_error = {$description}
```

Comment:
I'm a bit unsure about that "hack". That's here to forward the values of `ParserError` as lint. I'm not sure if there is a better way of doing that. If that's the case I'm happy to change it.

Reply:
It's fine until these parsing errors become translatable.